mozilla-release/mobile/android/base/strings.xml.in

```diff
@@ -183,6 +183,11 @@
   <string name="pref_category_notifications">&pref_category_notifications;</string>
   <string name="pref_category_notifications_summary">&pref_category_notifications_summary2;</string>
 
+  <!-- Cliqz start -->
+  <string name="pref_splash_screen_timeout">&pref_splash_screen_timeout;</string>
+  <string name="splash_screen_timeout_default">"4"</string>
```

Comment:
I suggest to put the value inside the code, as constant instead of getting from the resource